### ISSUE TITLE
edd_price_option_output filter $item_prop variable missing

### ISSUE THREAD

**Issue by davidsherlock** [easydigitaldownloads/themedd#244]
This may be exclusive to the 1.1 (or try 1.1 grid build), but the following override is included in Themedd by way of this function:

`themedd_edd_purchase_variable_pricing`

This function contains the same filter as the core EDD template function:

`$price_output = apply_filters( 'edd_price_option_output', $price_output, $download_id, $key, $price, $form_id );
`
However, the $item_prop variable is missing as per the original filter, resulting in issues with my sales plugin which filters this output.

**Comment by cklosowski** [comment]
@SDavisMedia If I'm not mistaken, the 1.1 branch is something we're using internally right? but it's a drastic move from how the public releases are built? Do we want to support this in the 1.1?

**Comment by SDavisMedia** [comment]
@cklosowski Correct, 1.1 is internal at the moment. It hasn't been released to anyone, though we've used it internally.

@davidsherlock Are you saying you built a plugin that depends on Themedd 1.1? Or are you using Themedd 1.1 on your own site, and only there you're seeing a conflict with your plugin?

**Comment by davidsherlock** [comment]
@SDavisMedia No, the theme (Themedd) overrides it's own version of the variable pricing options to allow for bootstrap styles of the checkboxes and the like. In doing so, a variable is left out of one of the original filters that I am using. Because I am building a theme agnostic plugin, I noticed this. It may be exclusive to the 1.1 (or grid 1.1) release.

**Comment by SDavisMedia** [comment]
@davidsherlock Got it. I'll milestone it for 1.1, which may not end up being a public release. I haven't decided yet. Thanks for pointing it out, but it shouldn't be a problem for your plugin since no one should be using Themedd 1.1.

**Comment by davidsherlock** [comment]
Thanks @SDavisMedia, just wanted to give you guys a heads up going forward in-case 1.1 is released publicly.

**Comment by robincornett** [comment]
Note that this is causing a fatal error on the downloads archive when All Access is active and a product has variable pricing. The problem is that All Access is definitely requiring the sixth parameter, but [Themedd is not including it](https://easydigitaldownloads.local/downloads/), probably because of proactively disabling schema.

@ashleyfae wanted to flag you on this comment:
> 
> 
> I'll milestone it for 1.1, which may not end up being a public release. I haven't decided yet. Thanks for pointing it out, but it shouldn't be a problem for your plugin since no one should be using Themedd 1.1.

Based on team call last week, I'm wondering if we should consider leaving Themedd at 1.0.8 and archiving it otherwise? Using the 1.1 version from before the master merge for internal projects? Looking through 1.1 overall I'm seeing quite a bit of work that would still need to be done.
